### DISCUSSION TITLE
Is there a setting for hiding window borders/edges when maximized?

### DISCUSSION THREAD

**Discussion by azcn2503** [yshui/picom#692] [Q&A]
Hi all,

One thing holding me back from using picom regularly is this very "first world problem" that causes me to inadvertently resize a window instead of scroll it, due to the scrollbar and window border being interactive at the same time. I spend an awful lot of time dragging scrollbars from the screen edge, but with picom I end up accidentally resizing windows.

I notice that XFCE's own compositor has the option to hide window borders when a window is maximized (also works when a window is snapped to the edge of the screen), and Mutter behaves in the same way.

I couldn't find a configuration option for this in my picom.conf. I am hoping that somebody can point me in the right direction. Or, if it doesn't exist yet, then I'd be happy to vote on an issue, if that's a thing!

Many thanks in advance.

**Comment by tryone144** [comment]
I am not sure whether I understand you correctly but window borders are a WM thing. Window decorations (like borders) are added and managed by the window manager.
Does xfwm not offer said option when using an external compositor?

**Reply by TiZ-HugLife** [reply]
xfwm4 does offer this option, and it works regardless of whether or not its own compositor is active, because as you said, window borders are a WM thing.

**Reply by domichel** [reply]
> Window decorations (like borders) are added and managed by the window manager.

It is why I like fvwm. It have no GUI for its configuration, but as fvwm can be seen as a toolkit for the Xlib, I can configure it as I want, the only limitation being my imagination and my time.
